tests/plugins/wysiwygarea/manual/detached.html

```diff
@@ -0,0 +1,48 @@
+<button id="toggleButton1">Toggle</button>
+<button id="setButton">Set data</button>
+
+<div>
+	<div id="editor">
+		<p>Lorem ipsum dolor sit amet.</p>
+	</div>
+</div>
+
+
+<div id="fakedId"></div>
+<div id="fakedId1"></div>
+
+
+<script>
+	var editorElement = document.getElementById("editor");
+
+	var textArea = document.createElement("textarea");
+
+	editorElement.appendChild(textArea);
+
+
+	var e = CKEDITOR.replace( textArea, {
```

Comment:
If possible, meaningful variable names are easier to understand when they are referred to later, so `editor` > `e`.